### PR TITLE
Add Thwaites-2020-98

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [x] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
Related to #1071. I additionally added a few new concept sets. The lists with unmapped left-over concepts will be linked in the next PR.

### CODE REVIEW

**Comment by Kristina-Pianykh** [comment]
Could I ask you to moderate this PR, @Schweikhard, if you have time?

**Comment by Schweikhard** [comment]
Sure! Suggesting @LinguList and @AnnikaTjuka as reviewers, if they have time.

**Review by LinguList — APPROVED** [review]

File: concepticondata/conceptlists/Thwaites-2020-98.tsv [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Thwaites-2020-98-27	27	injure	transitive	44	INJURE
+Thwaites-2020-98-28	28	irrigate	transitive	3078	IRRIGATE
+Thwaites-2020-98-29	29	knead	transitive	274	KNEAD
+Thwaites-2020-98-30	30	lick	transitive	319	LICK
+Thwaites-2020-98-31	31	nourish	transitive		
+Thwaites-2020-98-32	32	pacify	transitive		
+Thwaites-2020-98-33	33	pollinate	transitive		
+Thwaites-2020-98-34	34	promote	transitive		
+Thwaites-2020-98-35	35	propel	transitive		
+Thwaites-2020-98-36	36	raise	transitive	1317	RAISE
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Thwaites-2020-98.tsv:37]: We have "to raise, lift (tr.)" mapped to LIFT in a couple of lists and other forms of "raise" are mapped to RAISE (BRING UP). Is there maybe more information on the words in the study?

File: concepticondata/conceptlists/Thwaites-2020-98.tsv [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Thwaites-2020-98-40	40	scold	transitive	299	SCOLD
+Thwaites-2020-98-41	41	sprinkle	transitive		
+Thwaites-2020-98-42	42	stitch	transitive		
+Thwaites-2020-98-43	43	stow	transitive		
+Thwaites-2020-98-44	44	strangle	transitive		
+Thwaites-2020-98-45	45	terrify	transitive		
+Thwaites-2020-98-46	46	trounce	transitive		
+Thwaites-2020-98-47	47	underline	transitive		
+Thwaites-2020-98-48	48	vanquish	transitive		
+Thwaites-2020-98-49	49	whack	transitive	353	STRIKE
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Thwaites-2020-98.tsv:50]: I wouldn't map this since a closer synonym would be HIT.

**Review by AnnikaTjuka — COMMENTED** [review]
Looks good! I had just some minor comments.

File: concepticondata/conceptlists/Thwaites-2020-98.tsv (hunk 1 above)
Comment (Kristina-Pianykh) [reply concepticondata/conceptlists/Thwaites-2020-98.tsv:37]: Unfortunately, it's not specified. I've also checked now against the SUBTLES_UK database where it was taken from. So I'd rather unmap it then.

**Review by AnnikaTjuka — APPROVED** [review]
Thanks for considering my comments!

**Comment by Schweikhard** [comment]
Thanks for the reviews! Feel free to merge.

**Pull request merged**